website/docs/r/waf_byte_match_set.html.markdown

```diff
@@ -50,7 +50,8 @@ The following arguments are supported:
   e.g., `CONTAINS`, `CONTAINS_WORD` or `EXACTLY`.
   See [docs](http://docs.aws.amazon.com/waf/latest/APIReference/API_ByteMatchTuple.html#WAF-Type-ByteMatchTuple-PositionalConstraint)
   for all supported values.
-* `target_string` - (Optional) The value that you want to search forE.g., `HEADER`, `METHOD` or `BODY`.
+* `target_string` - (Optional) The value that you want to search for.
+  e.g., `badrefer1`.
   See [docs](http://docs.aws.amazon.com/waf/latest/APIReference/API_ByteMatchTuple.html#WAF-Type-ByteMatchTuple-TargetString)
```

Comment:
Looks like this link was out of date as well. Unfortunately, there's not an anchor link to go straight to this header 🙁 

```suggestion
  See [docs](https://docs.aws.amazon.com/waf/latest/APIReference/API_waf_ByteMatchTuple.html)
```

Reply:
Yes! Very much so thanks.